Assets/Scripts/CountDownTimer.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
Remove unused using statements. System.Collections and System.Collections.Generic